src/python/pants/core/util_rules/environments.py

```diff
@@ -303,22 +318,25 @@ class RemoteEnvironmentTarget(Target):
     )
     help = softwrap(
         """
-        Configuration of a remote execution environment used for building your code, including the
-        environment-sensitive options, which include environment variables and
-        search paths, used by Pants to execute processes in this remote environment.
+        Configuration of a remote execution environment used for building your code.
+
+        Environment configuration includes platform properties and a fallback environment, as well
```

Comment:
May be helpful to say those field aliases. Otherwise it's not clear what is from a subsystem override

Reply:
Are users likely to see this help text out of context with the valid fields for the target?